gateway/mw_auth_key.go

```diff
@@ -145,6 +146,7 @@ func (k *AuthKey) ProcessRequest(_ http.ResponseWriter, r *http.Request, _ inter
 	case apidef.AuthToken, apidef.UnsetAuth:
 		ctxSetSession(r, &session, updateSession, k.Gw.GetConfig().HashKeys)
 		k.setContextVars(r, key)
+		ctxSetSpanAttributes(r, k.Name(), otel.APIKeyAttribute(key))
```

Comment:
are we exposing here the key? I think we should avoid the key to be displayed in any system unless hashed or blurred

Reply:
On Tyk Pump we are sending the raw key - I'll ask around. Good question!